src/LuaAST.ts

```diff
@@ -99,37 +99,22 @@ export type Operator = UnaryOperator | BinaryOperator;
 
 export type SymbolId = number;
 
-export interface TextRange {
-    line?: number;
-    column?: number;
-}
-
-export interface Node extends TextRange {
+export interface Node {
     kind: SyntaxKind;
     parent?: Node;
+    tsOriginal?: ts.Node;
```

Comment:
I already discussed this with @tomblind a while ago. I think it is better not to keep TS types in the AST, at least from a design point of view.